gulpfile.js

```diff
@@ -0,0 +1,13 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+
+gulp.task('styles', function() {
+    gulp.src('sass/**/*.scss')
+        .pipe(sass().on('error', sass.logError))
+        .pipe(gulp.dest('./app/static/css'));
+});
+
+//Watch task
+gulp.task('default',function() {
+    gulp.watch('sass/**/*.scss',['styles']);
```

Comment:
Nice job! Just a couple of very minor nits. Could you add a space between `//` and the `W`. `// Watch task`. Could also add a space between parameters so that it is `('default', function() {` and `('sass/**/*.scss', ['styles'])`